src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -45,55 +55,97 @@ import {
+import { SCOPE_TYPE } from '@/store/modules/dashboard/type';
 import { FAVORITE_TYPE } from '@/store/modules/favorite/type';
 
 import FavoriteButton from '@/common/modules/favorites/favorite-button/FavoriteButton.vue';
 
 interface DashboardBoardListProps {
+    scopeType: ScopeType;
     fieldTitle: string;
     // TODO: implementation
-    dashboardList: any[];
+    dashboardList: DashboardItem[];
 }
 
 export default defineComponent<DashboardBoardListProps>({
     name: 'DashboardBoardList',
     components: {
-        PLabel, FavoriteButton, PBoard, PFieldTitle,
+        PPagination,
+        PLabel,
+        FavoriteButton,
+        PBoard,
+        PFieldTitle,
     },
     props: {
+        scopeType: {
+            type: String as PropType<ScopeType>,
+            default: undefined,
+        },
         fieldTitle: {
             type: String,
             default: undefined,
         },
         dashboardList: {
-            type: Array as PropType<any[]>,
+            type: Array as PropType<DashboardItem[]>,
             default: () => [],
         },
     },
     setup(props) {
         const state = reactive({
-            boardItemIconButtonSets: [
-                {
-                    iconName: 'ic_edit',
-                    eventAction: () => console.log('edit!'),
-                },
-                {
-                    iconName: 'ic_duplicate',
-                    eventAction: () => console.log('dup!'),
-                },
-                {
-                    iconName: 'ic_trashcan',
-                    eventAction: () => console.log('delete!'),
-                },
-            ],
-            dashboardListByBoardSets: computed(() => props.dashboardList.map((d) => ({ ...d, iconButtonSets: state.boardItemIconButtonSets }))),
+            currentPage: 1,
+            dashboardScopeType: computed(() => (props.scopeType === SCOPE_TYPE.DOMAIN ? 'domain' : 'project')),
+            dashboardListByBoardSets: computed(() => props.dashboardList
+                .slice((state.currentPage - 1) * 10, state.currentPage * 10)
+                .map((d) => (
+                    {
+                        ...d,
+                        iconButtonSets: convertBoardItemButtonSet(d[`${state.dashboardScopeType}_dashboard_id`]),
+                    }
+                ))),
         });
 
+        const convertBoardItemButtonSet = (dashboardId) => [
+            {
+                iconName: 'ic_edit',
+                eventAction: () => console.log('edit!'),
+            },
+            {
+                iconName: 'ic_duplicate',
+                eventAction: () => console.log('dup!'),
+            },
+            {
+                iconName: 'ic_trashcan',
+                eventAction: () => SpaceConnector.clientV2.dashboard[`${state.dashboardScopeType}Dashboard`].delete({
+                    [`${state.dashboardScopeType}_dashboard_id`]: dashboardId,
+                }),
```

Comment:
Is this instant code?
If not, I think you need to create a separate async function.

Reply:
Oh that's the part I missed. Thanks for finding it.